Very Small Example doesn't build on MacOS High Sierra (Pointer_prn__SDL_window & Pointer_prn_SDL_Renderer)

When I try to build the very small sdl example on the front page, I get a few errors concerning the following functions:

Pointer_prn__SDL_window

Pointer_prn__SDL_Renderer

clang told me that there were some similar functions, and changing all references to the above functions to:

Pointer_copy_SDL_window
Pointer_copy_SDL_Render

fixed it and I was able to build successfully. I'm not sure if the prn versions are supposed to be there and are missing forward declarations or if the carp compiler is supposed to be generating the copy versions. Either the forward declarations for the prn variants should be added to the output c file or they should be changed to use the copy versions

Huh, I get this too:

```
/main.c:9474:10: error: implicit declaration of function 'Pointer_prn__SDL_Window' is invalid in C99
      [-Werror,-Wimplicit-function-declaration]
  temp = Pointer_prn__SDL_Window(p->window); 
```

--- a/core/String.carp
+++ b/core/String.carp
@@ -462,5 +462,4 @@
 (defmodule Pointer
   (register str (Fn [(Ptr a)] String) "Pointer_str")
   (implements str Pointer.str)
-  (defn prn [x] (Pointer.str x))
 )

